src/drawing.rs

```diff
@@ -541,6 +542,47 @@ impl Drawing {
     pub fn poll_event(&mut self) -> Option<Event> {
         self.window.borrow_mut().poll_event()
     }
+
+    /// Saves the current drawings in SVG format at the location specified by `path`.
+    ///
+    /// ```rust
+    /// use turtle::{Turtle, Color};
+    ///
+    /// fn main() {
+    ///     let mut turtle = Turtle::new();
+    ///     turtle.drawing_mut().set_background_color("pink");
+    ///
+    ///     for i in 0..36 {
+    ///         let base_color: Color = if i % 2 == 0 {
+    ///             "red".into()
+    ///         } else {
+    ///             "white".into()
+    ///         };
+    ///         turtle.set_fill_color(base_color.with_alpha(1.0 - i as f64 / 54.0));
+    ///         turtle.begin_fill();
+    ///         square(&mut turtle);
+    ///         turtle.end_fill();
+    ///         turtle.right(10.0);
+    ///     }
+    ///    turtle.hide();
+    ///    turtle.drawing().save_svg("squares.svg");
+    /// }
+    ///
+    /// fn square(turtle: &mut Turtle) {
+    ///     for _ in 0..4 {
+    ///         turtle.forward(200.0);
+    ///         turtle.right(90.0);
+    ///     }
+    /// }
+    /// ```
+    ///
+    /// This will produce the following image in the current directory under the name `squares.svg`:
+    ///
+    /// #TODO: substitute actual link to squares.svg
+    /// ![squares](link to docs/assets/images/docs/squares.svg)
```

Comment:
I uploaded `squares.svg`, so you can use this URL: https://raw.githubusercontent.com/sunjay/turtle/master/docs/assets/images/docs/squares.svg?sanitize=true

Note the `sanitize=true` part at the end of the URL is [required](https://stackoverflow.com/questions/13808020/include-an-svg-hosted-on-github-in-markdown).